Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PI.mo

```diff
@@ -0,0 +1,70 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.Amigo;
+block PI "Identifies the parameters of a PI controller"
```

Comment:
"Identify parameters of the PI controller"

Reply:
Eliminated all the third-person singular